### ISSUE TITLE
The website and convention could (should?) mention CMIP and ESGF

### ISSUE THREAD

**Issue by jypeter** [cf-convention/cf-convention.github.io#256]
How strange, I have just noticed that **there is absolutely no mention of CMIP or ESGF in the CF convention**, not even in:
- the [Goals](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.10/cf-conventions.html#_goals)
- or the [References](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.10/cf-conventions.html#_bibliography)
- the [CF Standard Name Table](http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html)

Am I missing something? I guess the strong links between CMIP and CF are so obvious to our community that we failed to include this in the convention :-)

It could be useful to also have some link(s) to the [CMIP6 Data Request](https://cmip6dr.github.io/Data_Request_Home/), at the very least a link from the [CF Standard Name Table](http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html) to the [searchable list of variables in the request](http://clipc-services.ceda.ac.uk/dreq/mipVars.html)

**Comment by JonathanGregory** [comment]
Dear @jypeter 

Thanks for your comment. In recent years the needs of CMIP and ESGF have led to enhancements of the CF convention and many new standard names but CF predates both of them, doesn't depend on them, and serves many other communities too. CF does have a special relationship to the World Climate Research Programme (WCRP) Working Group on Coupled Modelling (WGCM). This is mentioned on the website under [Governance](http://cfconventions.org/governance.html) in the [CF constitution](http://cfconventions.org/constitution.html). WGCM supervises CMIP, but WGCM has wider aims than CMIP.

In view of this wider perspective, I don't feel it would be appropriate to make a special mention of CMIP or ESGF in the CF convention document, or indeed of any of the other projects where CF is used. If people would find it useful, however, we could compile and maintain on the website a list of projects which use CF.  Do you think that would help? What would you particularly like to achieve in making a connection between CF and CMIP, ESGF or the data request, I wonder?

Best wishes

Jonathan

**Comment by jypeter** [comment]
You are right, we don't really have to mention _CMIP_ in the convention itself, but having a link to **a page listing projects successfully using CF** in the documentation and in the [Quick links](http://cfconventions.org/) section of the home page would be useful and interesting (I'd like to know the other communities using CF, and see if they use tools I don't know of yet)

**Comment by JonathanGregory** [comment]
Following this suggestion by @jypeter, anyone is welcome to add comments to this issue listing projects which _use_ CF i.e.. depend on it, in some way. If we build up a substantial list, we can put them on a web page on the CF website. Note, this is different from projects which _support_ CF i.e. help others to use it; we already have a [list of software which supports CF](http://cfconventions.org/software.html).

On the list we so far have

* [Coupled Model Intercomparison Project](https://www.wcrp-climate.org/wgcm-cmip) (CMIP)
* [Input datasets for Model Intercomparisons Project](https://esgf-node.llnl.gov/search/input4MIPs) (input4MIPs)
* [Observations for Model Intercomparisons Project](https://esgf-node.llnl.gov/projects/obs4mips) (obs4MIPs)

I'm not sure that ESGF itself depends on CF.

Jonathan

**Comment by davidhassell** [comment]
There are also operational products that rely on CF, might be good to grow a list of those, too?

**Comment by larsbarring** [comment]
Cf. recent [discussion](https://github.com/orgs/cf-convention/discussions/322).

**Comment by larsbarring** [comment]
There is now a first first draft for a web page based on discussions in issue #256 and in  [Discussions #322](https://github.com/orgs/cf-convention/discussions/322).

Comments and input are welcome (and even necessary) to move this forward.

**Comment by ethanrd** [comment]
Hi Lars @larsbarring - To gather the widest range of projects possible, I think the title of the page should be simplified to something like "Projects and Activities that Use the CF Metadata Conventions".

**Comment by jypeter** [comment]
V 

On August 15, 2024 8:29:11 PM GMT+02:00, "Antonio S. Cofiño" ***@***.***> wrote:
>Closed #256 as completed via #503.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/cf-convention/cf-convention.github.io/issues/256#event-13900351545
>You are receiving this because you were mentioned.
>
>Message ID: ***@***.***>
-- 
JYP @ SGS 10e

**Comment by larsbarring** [comment]
I am reopening this as there were some stylistic improvements to be made in PR #503 (that now published). These have now been implemented and the new PR #519 is ready to go as it has been reviewed after which a couple of minor edits have been added. 

Could someone please have a quick final look and then merge (if nothing more surfaces).  ping @sadielbartholomew, @cofinoa 

Thanks,
Lars

**Comment by larsbarring** [comment]
Creating this page increasingly seems like jinxed work. After the recent improvements the acronym table at the bottom is mangled. I do not know why, but I will start a test PR shortly.

**Comment by sadielbartholomew** [comment]
Sorry Lars, I should have spotted that - I checked the formatting was OK after my first review, so it probably got botched after that somehow since I only did a final sanity check on your following commits.

**Comment by larsbarring** [comment]
I managed to do a test build using the usual gh workflow of the suggested change and it was indeed the lack of an empty line at the bottom of the table that was the culprit.
